client/lib/jetpack/wpcom-upsell-controller.tsx

```diff
@@ -12,9 +13,14 @@ export default function upsellSwitch(
 		const getState = context.store.getState;
 		const siteId = getSelectedSiteId( getState() );
 		const isJetpack = isJetpackSite( getState(), siteId );
+		const isAtomic = isSiteAutomatedTransfer( getState(), siteId );
 
-		if ( ! isJetpack && ! isEnabled( 'jetpack-cloud' ) && context.primary ) {
-			context.primary = <BusinessATSwitch UpsellComponent={ UpsellComponent } />;
+		if ( ! isEnabled( 'jetpack-cloud' ) && context.primary ) {
```

Comment:
I'm seeing the upsell for Atomic Business and Pro sites. Looks like `isEnabled( 'jetpack-cloud' )` is returning `false`.

![atomicpro-issue](https://user-images.githubusercontent.com/140841/174857595-e94697d5-73c2-4fa7-b5e7-596e46f5fead.png)

Reply:
Oh good catch! The `wpcom-upsell-controller` naming was misleading, and I thought this was covering only cases where the site is lacking the needed plan, but apparently this logic handles the Atomic transfers and shouldn't run if `isAtomic` is true. 

So I reverted these changes, renamed the file for clarity, and took a different approach to correctly display the upsell when needed.